src/celeritas/ext/GeantImporter.cc

```diff
@@ -237,34 +239,88 @@ import_particles(GeantImporter::DataSelection::Flags particle_flags)
     return particles;
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Return a populated \c ImportIsotope vector.
+ */
+std::vector<ImportIsotope> import_isotopes()
+{
+    auto const& g4isotope_table = *G4Isotope::GetIsotopeTable();
+    CELER_EXPECT(!g4isotope_table.empty());
+
+    std::vector<ImportIsotope> isotopes;
+    for (auto idx : range(g4isotope_table.size()))
+    {
+        auto const& g4isotope = *g4isotope_table[idx];
```

Comment:
I think you may want to get the pointer first and then check if it's null. The way the isotope table works is if you `delete` an isotope, then the corresponding table entry is `nullptr`. So you should really be pre-sizing `isotopes`, getting the pointer, skipping the loop if it's null, and assigning to `ImportIsotope& isotope = isotopes[idx]`.